### PR TITLE
[Feature] #90 Make tabbarUI

### LINKED ISSUE #90: [Feature] Make tabbarUI
# 유저는 장비 사전과 에디터 콘텐츠 화면을 확인할 수 있다.
- Tabbar UI를 구성한다

## 🤔 Completion conditions

- [ ] Can switch EditorMain, DictionaryMain

### PR DESCRIPTION
**Pull request by Gobans**
# Issue Number
🔒 Close #90 

## Changes

- Changed files
campair/campair.xcodeproj/project.pbxproj
campair/campair/Application/Assets.xcassets/DictionaryTabbarImage.imageset/Contents.json
campair/campair/Application/Assets.xcassets/DictionaryTabbarImage.imageset/DictionaryTabbarImage.svg
campair/campair/Application/Assets.xcassets/EditorTabbarImage.imageset/Contents.json
campair/campair/Application/Assets.xcassets/EditorTabbarImage.imageset/EditorTabbarImage.svg
campair/campair/Application/CampairApp.swift
campair/campair/View/TabbarView.swift

## New features

image

- TabbarUI

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

**Review by mijinc — APPROVED** [review]
GGOODD

File: campair/campair/View/TabbarView.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    var body: some View {
+        TabView {
+            EditorMainView()
+                .tabItem {
+                    Image(uiImage: editorIcon!)
+                    Text("에디터")
+                }
+            DictionaryMainView()
+                .tabItem {
+                    Image(uiImage: dictionaryIcon!)
```
Comment (sustainable-git) [inline campair/campair/View/TabbarView.swift:22]: Forced unwrapping here is unnecessary here.
There is an initializer that can get image from an image's name.
`Image(_ named: )`
And, you don't need 11~12 lines also.

**Review by sustainable-git — CHANGES_REQUESTED** [review]
The tab bar icon looks TIGHT though

File: campair/campair/View/TabbarView.swift (hunk 1 above)
Comment (rriver2) [reply campair/campair/View/TabbarView.swift:22]: I said to goban ! He will fix it !

**Pull request merged**